meson.build

```diff
@@ -155,12 +155,17 @@ add_project_arguments(
 
 # Common program references.
 prog_python = import('python').find_installation('python3')
+prog_env = find_program('env')
+prog_git = find_program('git')
+prog_srec_cat = find_program('srec_cat')
```

Comment:
These two are unused?

Reply:
srec_cat is used (in `sw/device/meson.build`), the `prog_git` is indeed unused and I just removed it.